HandleNHNotifyMessageCallActivityQueue/handler.ts

```diff
@@ -0,0 +1,95 @@
+import { flow, pipe } from "fp-ts/lib/function";
+import * as E from "fp-ts/Either";
+import * as TE from "fp-ts/TaskEither";
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import { TelemetryClient } from "applicationinsights";
+import { errorsToError } from "../IsUserInActiveSubsetActivity/handler";
+import {
+  buildNHService,
+  getNotificationHubPartitionConfig,
+  NotificationHubConfig
+} from "../utils/notificationhubServicePartition";
+import { toSHA256 } from "../utils/conversions";
+import { notify } from "../utils/notification";
+import { NhNotifyMessageRequest } from "../utils/types";
+import {
+  Failure,
+  throwTransientFailure,
+  toPermanentFailure,
+  toTransientFailure
+} from "../utils/errors";
+
+export type NhNotifyMessageResponse = Promise<
+  Failure | { readonly kind: string }
+>;
+
+export const handle = (
+  inputRequest: unknown,
+  legacyNotificationHubConfig: NotificationHubConfig,
+  notificationHubConfigPartitionChooser: ReturnType<
+    typeof getNotificationHubPartitionConfig
+  >,
+  fiscalCodeNotificationBlacklist: ReadonlyArray<FiscalCode>,
+  telemetryClient: TelemetryClient
+): NhNotifyMessageResponse =>
+  pipe(
+    inputRequest,
+    NhNotifyMessageRequest.decode,
+    E.mapLeft(flow(errorsToError, toPermanentFailure)),
+    TE.fromEither,
+    TE.bindTo("request"),
+    TE.bind(
+      "nhConfig",
+      ({
+        request: {
+          message: { installationId },
+          target
+        }
+      }) =>
+        TE.of(
+          target === "legacy"
+            ? legacyNotificationHubConfig
+            : notificationHubConfigPartitionChooser(installationId)
+        )
+    ),
+    TE.chain(({ request: { message }, nhConfig }) =>
+      pipe(
+        { kind: "SUCCESS", skipped: true },
+        TE.fromPredicate(
+          () =>
+            fiscalCodeNotificationBlacklist
+              .map(toSHA256)
+              .includes(message.installationId),
+          () => "execute notify"
+        ),
+        TE.orElseW(() =>
```

Comment:
I have to say that I'm not a big fan of switching the standard flow to the left path because of `TE.fromPredicate`, maybe we could try to use `TE.swap` right after it.

Reply:
swapping do not help us in this case: after the `TE.fromPredicate` both right and left path could be succesful path.

Reply:
True, but I continue to dislike using `TE.fromPredicate` this way. 
Nevermind, it's just my opinion.